ReactAndroid/build.gradle

```diff
@@ -213,6 +213,8 @@ android {
     compileSdkVersion 23
     buildToolsVersion "23.0.1"
 
+    useLibrary 'org.apache.http.legacy'
```

Comment:
Woops, I thought this gradle file is just for testing. That was workaround of https://github.com/robolectric/robolectric/issues/1862 but I'll find some other way.